osu.Game/Screens/Edit/Setup/DesignSection.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Localisation;
+using osu.Game.Graphics.UserInterfaceV2;
+
+namespace osu.Game.Screens.Edit.Setup
+{
+    internal class DesignSection : SetupSection
+    {
+        private LabelledSwitchButton widescreenSupport;
+        private LabelledSwitchButton epilepsyWarning;
+        private LabelledSwitchButton letterboxDuringBreaks;
+
+        public override LocalisableString Title => "Design";
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            Children = new[]
+            {
+                widescreenSupport = new LabelledSwitchButton
+                {
+                    Label = "Widescreen support",
+                    Current = { Value = Beatmap.BeatmapInfo.WidescreenStoryboard }
+                },
+                epilepsyWarning = new LabelledSwitchButton
+                {
+                    Label = "Epilepsy warning",
+                    Description = "Recommended if the storyboard contains scenes with rapidly flashing colours.",
+                    Current = { Value = Beatmap.BeatmapInfo.EpilepsyWarning }
+                },
+                letterboxDuringBreaks = new LabelledSwitchButton
+                {
+                    Label = "Letterbox during breaks",
+                    Current = { Value = Beatmap.BeatmapInfo.LetterboxInBreaks }
+                }
```

Comment:
The ordering is different from current stable, I assume you used the ordering of old image from the wiki? Not sure when/why that changed.

<img width="312" alt="Screen Shot 2021-08-22 at 1 15 00 PM" src="https://user-images.githubusercontent.com/35318437/130368814-120f9122-8f8f-483e-89f6-2c00c0f5ce46.png">

Reply:
I was using the old mock-ups from #2434. I see no reason not to switch to the current order if that is somehow deemed preferred.